### PR TITLE
Gaantunes/mongodb exporter

### PR DESCRIPTION
**Pull request by gaantunes**
#### PR Description

Adding mongodb exporter. For now it`s referencing a fork of the exporter, as soon as they merge a PR upstream I will reference it.

#### PR Checklist

- [X] CHANGELOG updated 
- [X] Documentation added
- [X] Tests updated

### CODE REVIEW

File: pkg/integrations/mongodb_exporter/logruskit.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	}
+}
+
+type output struct {
+	l            log.Logger
+	tsFromLogrus bool
+}
+
+func (o output) Write(data []byte) (n int, err error) {
+	var ll line
```
Comment (mattdurham) [inline pkg/integrations/mongodb_exporter/logruskit.go:44]: Would love a bit more detail/comments on this translation function.

File: pkg/integrations/mongodb_exporter/mongodb_exporter.go [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	return "mongodb_exporter"
+}
+
+// CommonConfig returns the common settings shared across all configs for
+// integrations.
+func (c *Config) CommonConfig() config.Common {
+	return c.Common
+}
+
+// NewIntegration creates a new elasticsearch_exporter
```
Comment (mattdurham) [inline pkg/integrations/mongodb_exporter/mongodb_exporter.go:50]: elasticsearch -> mongodb

File: pkg/integrations/mongodb_exporter/logruskit.go (hunk 1 above)
Comment (gaantunes) [reply pkg/integrations/mongodb_exporter/logruskit.go:44]: The credit for this one is due to @colegam, he gave me this hint to consume the exporter without needing to change it`s Log signature.

File: pkg/integrations/mongodb_exporter/mongodb_exporter.go (hunk 2 above)
Comment (gaantunes) [reply pkg/integrations/mongodb_exporter/mongodb_exporter.go]: woops, corrected :)

**Review by mattdurham — APPROVED** [review]

**Pull request merged**